app/scripts/services/rowMethods.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .factory('rowMethods', function() {
```

Comment:
sounds good to me. a generic storage state service, or just for expand/collapse? i assume we may eventually want to save other state.